packages/block-library/src/query-title/edit.js

```diff
@@ -109,18 +132,33 @@ export default function QueryTitleEdit( {
 		titleElement = (
 			<>
 				<InspectorControls>
-					<PanelBody title={ __( 'Settings' ) }>
-						<ToggleControl
-							__nextHasNoMarginBottom
+					<ToolsPanel
+						label={ __( 'Settings' ) }
+						resetAll={ () =>
+							setAttributes( {
+								showSearchTerm: false,
```

Comment:
```suggestion
								showSearchTerm: true,
```

The default value of the `showSearchTerm` attribute is `true`.